Release_Process.md

```diff
@@ -0,0 +1,42 @@
+# Confidential Container Release Process
+
+## Process
+- Code Freeze
+- Issue created from [Release Check list template](https://github.com/confidential-containers/community/issues/new?assignees=&labels=&template=release-check-list.md&title=%5BRelease%5D+Check+list+for+v%3CTARGET_RELEASE%3E)
+- "[Release Owner](https://github.com/confidential-containers/community/wiki/Release-Team-Rota)" identifies owners for Check List Items
+- Check List Items completed
+- Release Complete
+- Removal of Code Freeze and release announced.
+
+Release Dates will be set mid week and no one should feel an expectation to work unsocial hours to meet a release date. Communication and expectation setting is important and can be used to involve more people from the community to address any Issues blocking release. From the point of code freeze the release is being built so the release date is an estimate. Releasing early or late along with ideas for improvements/changes for future releases will be reflected on after release.
+
+
+## Requirements to help with Release Process
+- Availability during the release period to complete Tasks 
+- **TODO** What access is needed/suggested?
```

Comment:
I think this will be critical to the enabling individuals without needing to micro-manage a bunch of different repositories. Do we have a list of permissions required, yet?

Reply:
:-) Not yet waiting for input here, @fitzthum mentioned about creating a group and we can ensure that has the permissions, then we could document the need to be added to that group?

Reply:
The process is that we now have an organization-wide team called `release-champions`. Anyone who is in this team will elevated privileges on all repos that are involved in a release (thanks to that team having the custom Release role for each of these repos). So from a process standpoint once we have identified the people helping with a release, we should add them to this team. We should also make sure that we remove anyone who is not working on the current release to minimize any permissions issues.

Reply:
Thanks, I will also highlight the need to ensure people are added/removed to the responsibilities of the "release Owner"

Reply:
Yeah since being in that group gives people superpowers we probably don't want to have anyone on the team when we aren't in the freeze window.